src/components/attendee/ProfileOverlay/ProfileOverlay.tsx

```diff
@@ -80,19 +80,12 @@ export const ProfileOverlay: React.FC<ProfileOverlayProps> = ({ profile }) => {
     control,
   });
 
-  const { fields: links, append: addLink, remove: removeLink } = useFieldArray({
+  const { fields: links, append: addLink } = useFieldArray({
     control,
     name: "profileLinks",
     shouldUnregister: true,
   });
 
-  const onDeleteLink = useCallback(
-    (i: number) => {
-      removeLink(i);
-    },
-    [removeLink]
-  );
-
```

Comment:
Why have you removed the `deletion` funcitonality?

Reply:
in the current version of profile links there's no delete mechanism via a button ( we can only delete if we erase link's url and save the form ), so there's no need for this function